SettingsList.py

```diff
@@ -1610,7 +1610,16 @@ def __init__(self, name, gui_text, min, max, default, step=1,
                     Removes the requirements for the Lens of Truth
                     in Spirit Temple.
                     '''},
-}
+    'Beehives with Bombs/Bombchus' : {
+        'name'    : 'logic_beehives_bombs',
+        'tags'    : ("Beehives",),
+        'tooltip' : '''\
+                    Puts breaking beehives with bombs or bombchus into logic.
+                    Beehives in generic grottos can be reached with just bombs.
+                    All other beehives would require bombchus.
+                    '''},
+    }
+
```

Comment:
After testing on all of the beehives I've come to the conclusion that using bombs on all beehives where bombs can reach is completely fair. Bombchus on the other hand, if there were just one of two beehives I would say it was totally fine, but there are so many of them, and while all of them are easy enough, they all differ slightly in position and timing. It's the sum total of all of those bombchu uses that I think goes too far.

So I suggest making this trick exclusive to using bombchus on beehives, and having using bombs be default logic.